.github/workflows/CICD.yml

```diff
@@ -624,6 +613,13 @@ jobs:
         case '${{ matrix.job.os }}' in
           macos-latest) brew install coreutils ;; # needed for testing
         esac
+        case '${{ matrix.job.os }}' in
+          *linux*)
+            sudo sed -i 's/:,/:runner name,/' /etc/passwd
```

Comment:
please add a comment explaining why you are doing that

Reply:
Done.